src/lib/controllers/Controller.js

```diff
@@ -1,15 +1,14 @@
+// @flow
+
+// todo annotate better
 export default class Controller {
-  bad(message) {
-    return {
-      data: {message},
-      status: 400,
-    };
-  }
+  bad = (message: string): Object => ({
+    data: { message },
+    status: 400,
+  });
 
-  ok(output) {
-    return {
-      data: output,
-      status: 200,
-    };
-  }
+  ok = (output?: Object): Object => ({
```

Comment:
can't figure out the best way to annotate return type.. I was thinking about Ok<TModel> or something like that, but this way we need to do class controller<TModel> right?